time_manager/time_manager.F90

```diff
@@ -2521,18 +2521,18 @@ end subroutine time_manager_init
 !> @brief Prints the given time_type argument as a time (using days, seconds and ticks)
 !!
 !> @note There is no check for PE number.
-subroutine print_time (Time,str,unit)
+subroutine print_time (Time,str,iunit)
 type(time_type)  , intent(in) :: Time !< Time that will be printed
 character (len=*), intent(in), optional :: str !< Character string that precedes the printed time
-integer          , intent(in), optional :: unit !< Unit number for printed output, defaults to stdout
+integer          , intent(in), optional :: iunit !< Unit number for printed output, defaults to stdout
```

Comment:
This is technically a breaking change cause existing code would have to be changed if they use the name of the optional argument explicitly. We can see if any issues come up with model testing but i doubt it'll be an issue.

Reply:
Would you recommend that I don't change names of optional variables?

Reply:
Yeah i think if this is the only one, we can keep the original `unit` name so it doesn't break anything. It might be used more commonly than i initially thought and probably better to not put breaking changes for this smaller release.